app/src/main/java/com/mobilestudio/developer/extentions/Observers.kt

```diff
@@ -0,0 +1,9 @@
+package com.mobilestudio.developer.extentions
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+
+fun <T> LifecycleOwner.observe(liveData: LiveData<T>, action: (t: T) -> Unit) {
+    liveData.observe(this, Observer { it?.let { t -> action(t) } })
+}
```

Comment:
It seems to me that this extension is no longer necessary in the update of the Kotlin version to 1.4, it already allows you to do the same as you are doing in this extension function